CompletedProcess(args=['python', 'temp_test_case.py'], returncode=1, stdout=b'', stderr=b'.......FF.......F..........F.FF..........F.....F.....F...F...FFFF.......F....F..........F....F.F....\n======================================================================\nFAIL: test15 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 102, in test15\n    self.assertEqual(patched_source(input_15), original_source(input_15))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test16 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 108, in test16\n    self.assertEqual(patched_source(input_16), original_source(input_16))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test23 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 150, in test23\n    self.assertEqual(patched_source(input_23), original_source(input_23))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test33 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 210, in test33\n    self.assertEqual(patched_source(input_33), original_source(input_33))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test35 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 222, in test35\n    self.assertEqual(patched_source(input_35), original_source(input_35))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test36 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 228, in test36\n    self.assertEqual(patched_source(input_36), original_source(input_36))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test46 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 288, in test46\n    self.assertEqual(patched_source(input_46), original_source(input_46))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test51 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 318, in test51\n    self.assertEqual(patched_source(input_51), original_source(input_51))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test57 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 354, in test57\n    self.assertEqual(patched_source(input_57), original_source(input_57))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test60 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 372, in test60\n    self.assertEqual(patched_source(input_60), original_source(input_60))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test64 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 396, in test64\n    self.assertEqual(patched_source(input_64), original_source(input_64))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test65 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 402, in test65\n    self.assertEqual(patched_source(input_65), original_source(input_65))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test66 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 408, in test66\n    self.assertEqual(patched_source(input_66), original_source(input_66))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test67 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 414, in test67\n    self.assertEqual(patched_source(input_67), original_source(input_67))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test74 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 456, in test74\n    self.assertEqual(patched_source(input_74), original_source(input_74))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test79 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 486, in test79\n    self.assertEqual(patched_source(input_79), original_source(input_79))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test89 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 546, in test89\n    self.assertEqual(patched_source(input_89), original_source(input_89))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test93 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 570, in test93\n    self.assertEqual(patched_source(input_93), original_source(input_93))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n======================================================================\nFAIL: test95 (__main__.TestFunctions)\n----------------------------------------------------------------------\nTraceback (most recent call last):\n  File "/home/khashayar/projs/det-gen/DET-Gen2/temp_test_case.py", line 582, in test95\n    self.assertEqual(patched_source(input_95), original_source(input_95))\nAssertionError: Lists differ: [9012] != []\n\nFirst list contains 1 additional elements.\nFirst extra element 0:\n9012\n\n- [9012]\n+ []\n\n----------------------------------------------------------------------\nRan 100 tests in 0.012s\n\nFAILED (failures=19)\n')